src/App.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import './App.css';
-import { Home, About, Contact, Navigation } from './components';
+import { Home, About, Contact } from './components';
+import ReactDOM from "react-dom";
```

Comment:
im not sure what happened on this project with the package dependencies, but to get this one to render i had to import the components all individually here like so: import Home from './components/Home';  import About from './components/About';  import Contact from './components/Contact';